### ISSUE TITLE
What is the difference between "coowner" and "guest"?

### ISSUE THREAD

**Issue by FireController1847** [elBukkit/PreciousStones#106]
Hi! Thanks for keeping this up to date, it really means a lot. It's one of my favorite plugins. Unfortunately, I've noticed that every fork so far doesn't keep a live copy of the wiki. However, a lot of changes are being made that are worth documenting.

My biggest question is the new requirement for specifying "coowner" vs "guest" on a protection field. What's the difference between choosing "coowner" over "guest"? What limitations will a "guest" have over an "coowner"? What would be equivalent to the command before the change, "coowner" or "guest"?

Thanks for your time and again thank you for keeping it updated :)

**Comment by xtomyserrax** [comment]
Hey, how are you? Happy for the plugin to help. Guest and Coowner have existed since the start of the plugin, the only problem is that noone knew.

I sadly dont know all the differences as I havent coded all guest and coowner permissions, but I know a few players have been complaining of which I first thought it was a bug until I saw this. To start, in the past when you did /ps allow, for default allowed players were co owners, not guests. If you wanted them to be guests you needed to add a weird -g as parameter which wasnt documented. What I did is to make mandatory to specify this so people start to know about this feature.

A coowner is able to destroy the field block, to add new members to the field, to remove membres from the field, to manage flags, and maybe more stuff I dont really know. So if you add a player as a guest, it wont be able to do that.

Wish I was clear. Did you prefer how it was before? How can we make it better? Thanks!
